runzi/automation/run_time_dependent_solution.py

```diff
@@ -103,16 +103,16 @@ def run(source_solution_ids, current_years, mre_enums, forecast_timespans, aperi
 if __name__ == "__main__":
 
     # If you wish to override something in the main config, do so here ..
-    WORKER_POOL_SIZE = 1
+    WORKER_POOL_SIZE = 9
     #USE_API =
 
     # #If using API give this task a descriptive setting...
     TASK_DESCRIPTION = """Crustal. Geodetic. TD. From LTB89. Final """
     
-    TASK_TITLE = "Crustal. Geodetic. TD. From LTB89. Final. 100yr. NZ-SHM22 aperiodicity"
+    TASK_TITLE = "Crustal. Geodetic. TD. From LTB98. 100yr. NZ-SHM22 aperiodicity"
```

Comment:
typo on this one (LTB98 -> LTB 89) - I only noticed it when testing the new Weak search feature

Reply:
this is a problem for search. I've got a new feature that specifies this type of stuff in a toml file rather than as part of the script itself which should make typos easier to catch. Doesn't eliminate the problem entirely, though